app/src/main/java/com/sdp/movemeet/Activity/ActivityDescriptionActivity.java

```diff
@@ -159,12 +158,17 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
                 FirebaseInteraction.logoutIfUserNonNull(fAuth, this);
                 break;
             case R.id.nav_start_activity:
+                Navigation.startActivity(this.navigationView);
                 break;
             case R.id.nav_chat:
                 Navigation.goToChat(this.navigationView);
                 break;
+            case R.id.nav_list_activities:
+                Navigation.goToListOfActivities(this.navigationView);
```

Comment:
You need to add a "finish()" after this navigation to avoid overlap of activities. I already add this to all other items in my PR

Reply:
Okay !